jenkins/bootstrap.py

```diff
@@ -864,11 +870,12 @@ def setup_magic_environment(job, call):
         os.getenv(WORKSPACE_ENV, os.getcwd()), '_artifacts')
 
     # also make the artifacts dir if it doesn't exist yet
-    if not os.path.isdir(os.environ[JOB_ARTIFACTS_ENV]):
+    if not os.path.isdir(get_artifacts_dir()):
         try:
-            os.makedirs(os.environ[JOB_ARTIFACTS_ENV])
+            os.makedirs(get_artifacts_dir())
         except OSError as exc:
-            logging.info('cannot create $WORKSPACE/_artifacts, continue : %s', exc)
+            logging.info(
+                'cannot create $WORKSPACE/_artifacts, continue : %s', exc)
```

Comment:
nit: this should also from `get_artifacts_dir()`

Reply:
/shrug